tests/exception-policy-defrag-01/test.yaml

```diff
@@ -8,7 +8,7 @@ args:
 - -k none
 # pretend pretend error in the first fragment
 - --simulate-packet-defrag-memcap=1
-- --set defrag.memcap-policy=drop-packet
+- --set defrag.memcap-policy=drop-flow
```

Comment:
Not sure if we'd rather just error out if someone uses an exception policy value that isn't allowed for the given policy.

But we'd still need to check this, for the case of `auto`in IPS mode, for the master switch.